homeassistant/components/comfoconnect/sensor.py

```diff
@@ -62,274 +66,274 @@
+        name="Exhaust fan duty",
+        native_unit_of_measurement=PERCENTAGE,
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_DUTY,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        sensor_id=SENSOR_TEMPERATURE_EXHAUST,
+        multiplier=0.1,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_HUMIDITY,
+        device_class=DEVICE_CLASS_HUMIDITY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust humidity",
+        native_unit_of_measurement=PERCENTAGE,
+        sensor_id=SENSOR_HUMIDITY_EXHAUST,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_SUPPLY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Supply airflow",
+        native_unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-plus",
+        sensor_id=SENSOR_FAN_SUPPLY_FLOW,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_EXHAUST,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust airflow",
+        native_unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_FLOW,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_BYPASS_STATE,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Bypass state",
+        native_unit_of_measurement=PERCENTAGE,
+        icon="mdi:camera-iris",
+        sensor_id=SENSOR_BYPASS_STATE,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_DAYS_TO_REPLACE_FILTER,
+        name="Days to replace filter",
+        native_unit_of_measurement=TIME_DAYS,
+        icon="mdi:calendar",
+        sensor_id=SENSOR_DAYS_TO_REPLACE_FILTER,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_POWER_CURRENT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Power usage",
+        native_unit_of_measurement=POWER_WATT,
+        sensor_id=SENSOR_POWER_CURRENT,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_ENERGY_TOTAL,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        name="Energy total",
+        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+        sensor_id=SENSOR_POWER_TOTAL,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_PREHEATER_POWER_CURRENT,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Preheater power usage",
+        native_unit_of_measurement=POWER_WATT,
+        sensor_id=SENSOR_PREHEATER_POWER_CURRENT,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_PREHEATER_ENERGY_TOTAL,
```

Comment:
I don't think we should change the key for the sensors unless really necessary. Changing the key will be a breaking change since the unique_id will change. Just changing the default name won't be a breaking change.

Reply:
Yeah, but the key needs to be configured in the yaml config for now, so I think it's best to keep this in sync with the actual sensor name.

Reply:
My recommendation is to not make any breaking changes until the integration is reworked to use config entries. We can migrate unique_id then and YAML will be deprecated.

Reply:
Okay, but this PR started as a simple change to make it work with the new energy feature, 

Is it okay to just change the names of the sensors and keep the unique id's then? I'm not sure if that's what balloob had in mind.

Reply:
Just changing names is ok.